content/sensu-go/5.0/reference/rbac.md

```diff
@@ -30,6 +30,7 @@ Sensu access controls apply to [sensuctl][2], the Sensu [API][19], and the Sensu
 Namespaces help teams use different resources (checks, handlers, etc.) within Sensu and impose their own controls on those resources.
 A Sensu instance can have multiple namespaces, each with their own set of managed resources.
 Resource names need to be unique within a namespace, but not across namespaces.
+Sensu agents can only belong to a single namespace.
```

Comment:
```suggestion
```

Reply:
To me this sentence feels disconnected from the rest of the paragraph, which says to me this is the wrong place for us to call it out. Consider adding a note to this affect in the `Namespaced resource types` section, following the table of resources.